staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go

```diff
@@ -40,7 +40,6 @@ func (oa OpenAPI) InstallV2(c *restful.Container, mux *mux.PathRecorderMux) (*ha
 	if err != nil {
 		klog.Fatalf("Failed to build open api spec for root: %v", err)
 	}
-	spec.Definitions = handler.PruneDefaults(spec.Definitions)
```

Comment:
Moved this line closer to usage in `genericapiserver.go` so that we can use the spec to generate SMD schemas before it is mutated. Happy to refactor however, though.